src/sonic-config-engine/tests/test_cfggen_platformJson.py

```diff
@@ -0,0 +1,76 @@
+from unittest import TestCase
+import subprocess
+import os
+import json
+import ast
+
+# Global Variable
+PLATFORM_CAP = "platform_output.json"
```

Comment:
This is unused, yet you refer to `platform_output.json` explicitly on line 69.

Also, `PLATFORM_CAP` is not a clear name for this constant. Suggest to rename `PLATFORM_CAPABILITIES_FILE`, `PLATFORM_OUTPUT_FILE` or similar.

Reply:
#done